webapp/client/src/components/FormHeaderParagraphs.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Intro = styled.p`
+  font-size: var(--text-medium);
+`;
+
+export default function FormHeaderParagraphs({ title, intros }) {
+  return (
+    <div>
+      <h1 className="my-4">{title}</h1>
+      {intros.map((intro, index) => (
+        <Intro key={`intro_${index}`}>{intro}</Intro>
+      ))}
+    </div>
+  );
+}
```

Comment:
I'd expect the FormHeaderParagraphs to only include the paragraphs, not only the title. so either we change the name or (my preferred way) set the title in the FormHeader directly.

Reply:
And, I imagine it to throw an error in eslint with the index as key? normally we have to explicitly tell eslint not to check that, right?

Reply:
> I'd expect the FormHeaderParagraphs to only include the paragraphs, not only the title. so either we change the name or (my preferred way) set the title in the FormHeader directly.

Ah, you mean that that FormHeader part would be only the paragraphs and the title would come from somewhere else? Basically having two or three simple components that could be combined?

The key is "intro_1". This should not throw an error and it did not.

Reply:
Not sure if that was a typo or if you got me wrong:

> Ah, you mean that that FormHeader part would be only the paragraphs

No. I meant that * FormHeaderParagraphs * only consists of the paragraphs (as the name suggests) and FormHeader then has <h1/> <FormHeaderParagraphs/> you know?

Reply:
I think I get it now. I moved the intro into its own component (former FormHeaderParagraphs). The FormHeader composes the FormHeader from the title and the FormIntro (paragraphs or a single intro)